test/fixtures/output/flink/statement/list-completed.golden

```diff
@@ -0,0 +1,4 @@
+          Creation Date         |         Name         |     Statement      | Compute Pool |  Status   |       Status Detail         
+--------------------------------+----------------------+--------------------+--------------+-----------+-----------------------------
+  2022-01-01 00:00:00 +0000 UTC | 22222222-2222-2222-2 | CREATE TABLE test; | lfcp-123456  | COMPLETED | SQL statement is completed  
```

Comment:
Out of curiosity: it looks like the backend doesn't care if a statement ends with a semi-colon. For the purposes of user-facing examples, should we recommend adding a semi-colon or not?
```suggestion
  2022-01-01 00:00:00 +0000 UTC | 22222222-2222-2222-2 | CREATE TABLE test; | lfcp-123456  | COMPLETED | SQL statement is completed  
```

Reply:
the backend doesn't care but the client recognises semicolons as statement terminator

Reply:
Sounds like we should recommend adding a semi-colon, in that case?